wasm/event-stubs/Earthfile

```diff
@@ -0,0 +1,29 @@
+VERSION 0.7
+
+# cspell: words wasi autodrop webassembly WORKDIR CFLAGS mexec
```

Comment:
`wasi` `webassembly` `WORKDIR` seem to be common word using across the project, might move them to the dict